### PR TITLE
Few updates to the README

### PR DESCRIPTION
**Pull request by rthallisey**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,17 +108,13 @@ spec:
 EOF
 ```
 
-Create an HCO CustomResource, which creates the KubeVirt CR, launching KubeVirt.
+Create an HCO CustomResource, which creates the KubeVirt CR, launching KubeVirt,
+CDI, Network-addons, and SSP.
 ```bash
 kubectl create -f deploy/hco.cr.yaml
```
Comment (rwsu) [inline README.md]: is -n required here? I couldn't see where the CR is deployed if -n is unspecified. Didn't see it in the default namespace.

**Comment by rthallisey** [comment]
ci test please

**Comment by rthallisey** [comment]
ci test please

**Comment by rthallisey** [comment]
ci test please

**Comment by rthallisey** [comment]
ci test please

**Pull request merged**